packages/markdown-support/src/index.ts

```diff
@@ -69,10 +69,10 @@ export async function renderMarkdown(content: string, opts?: MarkdownRenderingOp
     const vfile = await parser
       .use(raw)
       .use(rehypeCollectHeaders)
-      .use(rehypeCodeBlock())
+      .use([rehypeCodeBlock()])
       .use(rehypeStringify, { entities: { useNamedReferences: true } })
       .process(content);
-    result = vfile.contents.toString();
+    result = vfile.value.toString();
```

Comment:
This was a `vfile@5` breaking change: https://github.com/vfile/vfile/releases/tag/5.0.0

vfile was upgraded in `unified@10`

Reply:
Fixed! Thanks